Source/NSButtonCell.m

```diff
@@ -1674,7 +1628,9 @@ - (void) encodeWithCoder: (NSCoder*)aCoder
       buttonCellFlags.changeGray = [self cellAttribute: NSChangeGrayCell];
 
       // set these to zero...
-      buttonCellFlags.unused1 = 0; // 32;
+      buttonCellFlags.inset = 0; // 32;
```

Comment:
What does //32 mean in these three lines?

Reply:
I removed these, as I also don't understand what this is supposed to explain.